src/core/public/saved_objects/saved_objects_client.test.ts

```diff
@@ -35,11 +35,13 @@ import { SimpleSavedObject } from './simple_saved_object';
 import { httpServiceMock } from '../http/http_service.mock';
 
 describe('SavedObjectsClient', () => {
+  const updatedAt = new Date().toString();
```

Comment:
Why not use the ISO 8601 standard with `new Date().toISOString()`?

Reply:
sure